pkg/importer/tests-swagger/path-var-overwrite.sysl

```diff
@@ -12,7 +12,7 @@ testapp "Sample API" [package="package_foo"]:
         | API description in Markdown.
 
     /users/{id<:int}:
-        GET (metadata <: _param_metadata [~header, ~required, name="metadata"]):
+        GET (metadata <: string [~header, ~required, name="metadata"]):
```

Comment:
this should be `bool` https://github.com/anz-bank/sysl/blob/master/pkg/importer/tests-swagger/path-var-overwrite.yaml#L16